package.json

```diff
@@ -79,6 +79,9 @@
     "qunit-dom": "^2.0.0",
     "webpack": "^5.73.0"
   },
+  "resolutions" : {
+    "ember-modifier": "^3.2.7"
+  },
```

Comment:
Did we need to pin `ember-modifier` because of `ember-container-query` running an older version? If so, I think updating the latter addon to the latest version will allow us to run `ember-modifier@v4`. This can be done in a separate task.

Reply:
Indeed.

Upgrading `ember-container-query` to v4 was quite straight forward so I kept the changes in this PR instead of opening a separate one.